debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -383,11 +464,36 @@ public Duration snapshotLockTimeout() {
         return Duration.ofMillis(getConfig().getLong(SNAPSHOT_LOCK_TIMEOUT_MS));
     }
 
-    private static int validateColumnBlacklist(Configuration config, Field field, Field.ValidationOutput problems) {
-        String whitelist = config.getString(COLUMN_WHITELIST);
-        String blacklist = config.getString(COLUMN_BLACKLIST);
-        if (whitelist != null && blacklist != null) {
-            problems.accept(COLUMN_BLACKLIST, blacklist, "Column whitelist is already specified");
+    public String schemaExcludeList() {
```

Comment:
Do we indeed need to expose all of them? Ideally, access only would be via `getFilters()`.

Reply:
see the comment on the PostgresConnectorConfig. I moved and renamed them from there. I think it's architecture-wise the best way to have getters here and not access properties by config.getString(....) etc.

Reply:
Yes, that's better. But eventually, the raw filter properties shouldn't be exposed at all, rather only the Filters class should provide the access. But it seems you're saying that right now we still need to do this?

Reply:
That's still pending?

Reply:
I think we should do this in a future refactoring. maybe introduce a FiltersConfig in the `relational`?
What do you mean with "pending"? Is it okay like this or what do you think needs to be done? @gunnarmorling

Reply:
Thanks for logging that one. We can discuss this over there.